ouroboros-network/test/Test/Mux.hs

```diff
@@ -136,7 +137,7 @@ sduSizeMux :: (Monad m)
            -> m Word16
 sduSizeMux ctx = return $ sduSize ctx
 
-writeMux :: (MonadTimer m, MonadSTM m, Mx.ProtocolEnum ptcl)
+writeMux :: (MonadTime m, MonadSTM m, Mx.ProtocolEnum ptcl)
```

Comment:
Previously it wasn't possible to set the timestamp outside of the IO monad. With these changes it should be possible so you should update `writeMux` to do it.

Reply:
Done.